vagrant-serverspec.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |gem|
   #add onlly dependencies for winrm without nokogiri
   gem.add_runtime_dependency 'winrm', '~> 2.1', '>= 2.1'
   gem.add_runtime_dependency 'os', '~> 0.9.6'
+  gem.add_runtime_dependency 'rspec_html_formatter', '~> 0.3', '>= 0.3.1'
```

Comment:
I really like the generated html report and too bad the gem is not updated anymore. @vvchik Could you please take a look at this MR? I think it will be :ok:  to merge.

Reply:
No problem for me with this MR